api/src/test/resources/io/strimzi/api/kafka/model/KafkaUser.yaml

```diff
@@ -13,18 +13,18 @@ spec:
           type: topic
           name: my-topic
           patternType: prefix
-        operation: Read
+        operations: ["Read","Write"]
```

Comment:
Here and elsewhere ... are the quotes really needed? I don' think they should be. Also, maybe we should use here an encoding as list?

```yaml
operations:
- Read
- Write
```